CONTRIBUTING.md

```diff
@@ -0,0 +1,128 @@
+# How can I contribute?
+
+- You can fix the typos (if any) in any documentation of this repo.
+- You can check for any **unassigned** [issues](https://github.com/Real-Dev-Squad/feature-flag-backend/issues) and **comment** on that issue that you'd like to get that issue assigned to you. (Remember: Do not work on issues assigned to someone else and do not work on any issue without having it assigned to you.)
+- Create issue if you see any bug and then once you get approved from the admins, you can assign yourself and start working on it.
```

Comment:
[NIT] Also please mention that communicate the ETA for the task. It will make the things clear for people contributing

Reply:
Done